packages/app/obojobo-document-engine/src/scripts/oboeditor/components/editor-app.js

```diff
@@ -30,6 +30,7 @@ const { OboModel } = Common.models
 
 const XML_MODE = 'xml'
 const VISUAL_MODE = 'visual'
+const RENEW_LOCK_INTERVAL = 60000 * 4.9 // 4.9 minutes in milliseconds
```

Comment:
Is it possible to pull this from the new config variable `editLockExpireMinutes`?

Reply:
it'd be a lie to do so.  This gets compiled and thrown into js and the other is re-configurable at run time. We could change it so it's loaded from the server in the template launch.